aspnetcore/client-side/spa/intro/mermaid/sequenceDiagram.txt

```diff
@@ -0,0 +1,35 @@
+```mermaid
+sequenceDiagram
+participant Browser
+participant Proxy
+participant Server
+Browser->>Server: GET /
+alt Proxy is running
+  Server->>Browser: 301 Redirect <<Proxy-URL>>
+else Proxy is not running
+  Server->>Proxy: Launch
+  par Browser checks with server if the proxy is running
```

Comment:
@peterblazejewicz  what about replacing `par` with `check`? What do you suggest?